draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+    \end{itemize}
+   \item \textbf{\nth{2} Round:}
+    \begin{itemize}
+      \item  First wave of enemy clan armies attacks the towns. (see below)
+      \item  Each player who defended his own town, during this Round can hire secondary Hero for 5 \svg{gold}, but you still have to use the population token.
+    \end{itemize}
+   \item \textbf{\nth{3} Round:}
+    \begin{itemize}
+      \item At start of the Round, each player shows up one Tile (Far (II--III) Map Tile. This Tile you can reveal freely. (Not matter if Tile is from pool or map template.)
+    \end{itemize}
+   \item \textbf{\nth{4} Round:}
+    \begin{itemize}
+      \item Second wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+     \end{itemize}
+   \item \textbf{\nth{6} Round:}
+    \begin{itemize}
+      \item Third wave of enemy clan armies attacks the towns. (see below)
+      \item Each player who defended his own town, search (3) the \textit{Artifact Deck}.
+    \end{itemize}
+   \item \textbf{\nth{7} Round:}
+    \begin{itemize}
+      \item During this Round, Players can discard and draw Cards at the start of their turn twice rather than once.
+    \end{itemize}
+  \end{itemize}
+
+\subsection*{\MakeUppercase{Enemy's clans waves}}
+\begin{itemize}
+  \item If enemy's settlement is not conquered, then enemy clans armies are attacking your castle!
+  \item At the start of the each player's turn, the battle begins.
+  \item Before attack, you can use your building, spell book and population tokens, and use Might and Magic Cards, but you can't move with Hero.
+  \item Each player has to defend his own castle.
+  \item If you have citadel, \textit{Arrow tower} has decreased attack. The attack is for first/second/third wave as 2/3/4\svg{attack}.
+  \item The battle does not cost any extra movement points.
+  \item The strength of the siege army is determined by the table \textit{Enemy's Clans armies}.
+  \item If your main Hero is present in any of allied town (not settlement) and Clan wave attack your allied town where is no main Hero, you can participate on defending allied castle by using Might and Magic Cards (Simply: you defend your teammate's castle with their Units and your Hero's Cards).
+  \item If you defended your town, gain one \svg{experience}. It doesn't matter if the main Hero was in the fight.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Settlements defenders}}
+\begin{itemize}
+  \item The strength of the army in Settlement is determined by the table \textit{Enemy's Clans armies} (last row).
+  \item After draw an enemy army, Player can discard one enemy Unit Card and draw a new Unit of same Tier.
+  \item The battle does not cost any extra movement points.
+  \item In this battles, you has to use AI Cards.
+  \item After won battle, you gain one Level.
```

Comment:
```suggestion
  \item After winning a combat, gain one Level.
```
What exactly do you mean by that? Let's say your hero has 5 exp, meaning they're at level 2, need one experience to advance to level 3. On the other hand, having 4 exp would also advance them to level 3? Depending on the situation, the hero gets either 1 or 2 experience points?

Reply:
```suggestion
  \item After winning a combat, gain 2 \svg{experience}.
```
In that case, please make it precise. The original rule book is not reliable in this case.